packages/debug-utils/index.js

```diff
@@ -372,10 +373,12 @@ var DebugUtils = {
     }
     if (
       breakpoint.compilationId !== currentCompilationId ||
+      breakpoint.context !== currentInternalContext ||
```

Comment:
It kind of strikes me as a code smell that we expose the context here like this. Is there a way to provide an interface to the debugger to get the source more opaquely? I guess this goes back to your question of whether or not we should ensure source ID uniqueness... there might be value in doing that.

Reply:
Done! :)